common/views/components/IIIFViewerPrototype/ViewerSidebarPrototype.tsx

```diff
@@ -284,11 +281,10 @@ const ViewerSidebarPrototype: FunctionComponent<Props> = ({
           </AccordionItem>
         )}
       </div>
-      {searchService && itemViewerPrototypeWithSearch && (
```

Comment:
we still need the searchService check, as we don't want the search form to show if there is no way of searching the item

Reply:
Ah, good shout - is this right?